src/guides/v2.4/comp-mgr/trouble/current-search-engine.md

```diff
@@ -0,0 +1,57 @@
+---
+group: software-update-guide
+title: Your current search engine is not supported
+functional_areas:
+  - Upgrade
+---
+
+The `Your current search engine, <Engine Name>, is not supported. You must install a supported search engine before upgrading. See the System Upgrade Guide for more information.` error message indicates that Magento version you are upgrading from is configured to use a catalog search engine that is not supported in the Magento version you are upgrading to. This error means one of the following conditions are true on the down-level version of Magento:
```

Comment:
Consider separating the error message from the paragraph to improve readability. For example:

The following error message indicates that Magento version you are upgrading from is configured to use a catalog search engine that is not supported in the Magento version you are upgrading to:

```
Your current search engine, <Engine Name>, is not supported. You must install a supported search engine before upgrading. See the System Upgrade Guide for more information.
```

This error means one of the following conditions are true on the down-level version of Magento: